packages/jest-util/src/index.js

```diff
@@ -31,13 +32,28 @@ const createDirectory = (path: string) => {
   }
 };
 
+const clearDirectory = (path: string) => {
+  if (fs.existsSync(path)) {
+    fs.readdirSync(path).map(file => {
+      const filePath = `${path}/${file}`;
```

Comment:
This won't work on windows, use `path.join` or `path.resolve`

Reply:
Didnt even think of that. Will research.